### ISSUE TITLE
Once broken, Mr. Bag doesn't ever seem to get repaired, no matter the days passed.

### ISSUE THREAD

**Issue by Nextej** [loiste-interactive/Obenseuer-Issues#411]

**Comment by Zuntti** [comment]
Seems to work for me. It should take about one hour to sign appear and one day to be repaired

**Comment by Nextej** [comment]
Should you be on the map or outside of it?

**Comment by Zuntti** [comment]
Outside

**Comment by Nextej** [comment]
Checked it out, I got it broken on Mon 9:00 and slept to Wed 6:00 while being outside the map, pretty sure the last time I've tried it took even more day of testing.
[Player.log](https://github.com/loiste-interactive/Obenseuer-Issues/files/12269840/Player.log)

I'm also pretty sure that it worked without problems on a previous version just as expected, to be fixed after a day.

On the topic of it, perhaps a day until the bot will be fixed should be stated on the sign? So the player is not left entirely alone on this one (which otherwise may just cause them to reveal the safe number and reload from before breaking it down to keep the benefits).
